test/distributed/test_functional_api.py

```diff
@@ -394,9 +411,12 @@ def setUp(self):
         super().setUp()
         self._spawn_threads()
 
-    def test_all_reduce_tracing(self):
+    @run_with_both_funcol_impls_with_arg
+    def test_all_reduce_tracing(self, use_native_funcol):
+
         def allred(input):
-            return ft_c.all_reduce(input, "sum", group=[0, 1]) + 1
+            group = "0" if use_native_funcol else [0, 1]
```

Comment:
iirc the default group name is "default", is it still the case or it's actually "0"?